monkey/infection_monkey/credential_collectors/ssh_collector/ssh_credential_collector.py

```diff
@@ -5,12 +5,12 @@
 from common.event_queue import IAgentEventQueue
 from common.types import AgentID
 from infection_monkey.credential_collectors.ssh_collector import ssh_handler
-from infection_monkey.i_puppet import ICredentialCollector
+from infection_monkey.i_puppet import ICredentialsCollector
 
 logger = logging.getLogger(__name__)
 
 
-class SSHCredentialCollector(ICredentialCollector):
+class SSHCredentialCollector(ICredentialsCollector):
```

Comment:
For Mimikatz too
```suggestion
class SSHCredentialsCollector(ICredentialsCollector):
```
But I guess we could skip this since they're going to be migrated to plugins soon.

Reply:
+1 for skipping for now.